src/open_producten/producttypen/tests/api/test_link.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework.test import APIClient
+
+from open_producten.producttypen.models import Link, ProductType
+from open_producten.utils.tests.cases import BaseApiTestCase
+from open_producten.utils.tests.helpers import model_to_dict_with_id
+
+from ..factories import LinkFactory, ProductTypeFactory
+
+
+def link_to_dict(link):
+    link_dict = model_to_dict_with_id(link, exclude=["product_type"])
+    link_dict["product_type_id"] = link.product_type.id
+
+    return link_dict
+
+
+class TestProductTypeLink(BaseApiTestCase):
+
+    def setUp(self):
+        super().setUp()
+        product_type = ProductTypeFactory.create()
+        self.data = {
+            "naam": "test link",
+            "url": "https://www.google.com",
+            "product_type_id": product_type.id,
+        }
+        self.path = "/api/v1/links/"
+
+        self.link = LinkFactory.create(product_type=product_type)
+
+    def test_read_link_without_credentials_returns_error(self):
+        response = APIClient().get(self.path)
+        self.assertEqual(response.status_code, 401)
+
+    def test_create_link(self):
+        response = self.post(self.data)
```

Comment:
```suggestion
        response = self.client.post(self.path, self.data)
```

I think all the tests should be like this

`return self.client.post(self.path, data, format="json")` -> I think the format json is unnecessary

Reply:
I think you should also test that all fields are set properly here.

Reply:
And maybe test required fields, which is all. But the API should return field x is required, field y is required